src/H5FDros3.h

```diff
@@ -104,6 +112,20 @@ H5_DLL herr_t H5Pget_fapl_ros3(hid_t fapl_id, H5FD_ros3_fapl_t *fa_out);
  */
 H5_DLL herr_t H5Pset_fapl_ros3(hid_t fapl_id, const H5FD_ros3_fapl_t *fa);
 
+/**
+ * \ingroup FAPL
+ *
+ * \todo Add missing documentation
+ */
+H5_DLL herr_t H5Pget_fapl_ros3_token(hid_t fapl_id, size_t size, char *token);
```

Comment:
It would be great if we could have documentation for these API routines before this PR is merged.

Reply:
I have added doxygen-style documentation for the defines, struct and functions. Most likely, not perfect but at least there is something now.

Reply:
Thanks for going above and beyond and adding all the doxygen in this file!